thunderbird/processes/wps_generate_climos.py

```diff
@@ -190,7 +190,10 @@ def format_resolutions(self, resolutions):
         return list(set(resolutions))
 
     def collect_args(self, request):
-        climo = self.format_climo([climo.data for climo in request.inputs["climo"]])
+        if "climo" in request.inputs:
+            climo = self.format_climo([climo.data for climo in request.inputs["climo"]])
+        else:
+            climo = standard_climo_periods().keys()
```

Comment:
I'm not sure if this is possible, but I think we can set the default values when we define the `inputs` above.